svo (setup, misc, empty, funnies, dor).xml

```diff
@@ -728,9 +725,9 @@ if not svo.systemloaded then
   conf.ai_resetherbbal      = 2.5 -- normally at 1.6
   conf.ai_resetsalvebal     = 5
   conf.ai_resetmossbal      = 10  -- resets at 6
-  if svo.haveskillset('healing') then
-    -- resets at 2s for healing allies, near 4s for healing yourself, and offensive skills as inbetween
-    conf.ai_resethealingbal        = 7
+  if svo.haveskillset('zeal') then
+    -- resets at 3 for blessings, 4 for benediction 1.6-2.3 for affs
+    conf.ai_resetprayerbal        = 1.7
```

Comment:
1.7 seems super low? It means the balance will always reset at 1.7s as a failsafe, regardless of how long it actually takes

Reply:
This variable isn't actually referenced anywhere in the system...

Should probably add it to svo.lostbal_prayer() for the tempTimer time.